.evergreen/config.yml

```diff
@@ -356,7 +356,7 @@ tasks:
         - func: "run atlas data lake test"
 
     - name: "test-requireApiVersion"
-      tags: ["versioned_api"]
+      tags: ["versioned-api"]
```

Comment:
Should this test also use `TESTS: "versioned-api"` to select a subset of the PHPUnit test suite, or does it intentionally need to target the entire suite?

Reply:
The Versioned API spec calls for running the entire test suite against a server with `requireApiVersion` and authentication enabled to ensure that all commands declare an API version, as they can't be checked using command monitoring.